dayrepo/snippets/static/js/process_add.js

```diff
@@ -1,9 +1,35 @@
-let i = 1 ;
 const addForm = () => {
-  let content_area = document.getElementById("form_area");
+  if (isNaN(document.getElementById("count").innerHTML)){ 
+    alert("カウント値が異常です")
+    return
```

Comment:
セミコロンはつけるならつけるで統一しましょう〜
```suggestion
    return ;
```

他の addForm の中も全部お願いしますます